src/applications/financial-status-report/components/otherAssets/OtherAssetsSummary.jsx

```diff
@@ -22,16 +30,29 @@ const OtherAssetsSummary = ({
 
   useEffect(
     () => {
-      if (!gmtData?.isEligibleForStreamlined) return;
+      let isMounted = true;
+
+      function fetchData() {
+        const calculatedAssets = calculateTotalAssets(data);
+        if (isMounted) {
+          // Check here
+          setFormData({
+            ...data,
+            gmtData: {
+              ...gmtData,
+              assetsBelowGmt: calculatedAssets < gmtData?.assetThreshold,
+            },
+          });
+        }
+      }
 
-      const calculatedAssets = calculateTotalAssets(data);
-      setFormData({
-        ...data,
-        gmtData: {
-          ...gmtData,
-          assetsBelowGmt: calculatedAssets < gmtData?.assetThreshold,
-        },
-      });
+      if (gmtData?.isEligibleForStreamlined) {
+        fetchData();
+      }
+
+      return () => {
+        isMounted = false;
+      };
```

Comment:
This is the cleanup function that was added because of a console memory leak warning